dbt/tests/generic/test_is_null.sql

```diff
@@ -1,13 +1,23 @@
 -- Confirm that a column is null. Essentially, the opposite of the `not_null`
 -- test.
-{% test is_null(model, column_name, additional_select_columns=[]) %}
+{% test is_null(model, column_name, additional_select_columns=[], anti_join={}) %}
```

Comment:
I wish there were a way to support anti-joins without having to update each individual generic to support it, but unfortunately I think this is the best approach. If we decide to roll it out more widely, we could think about taking a similar approach to the one we took with `additional_select_columns`, namely factoring out the shared logic for the param into a dedicated macro.